artifactory/cli.go

```diff
@@ -2326,6 +2341,33 @@ func transferConfigCmd(c *cli.Context) error {
 	return nil
 }
 
+func transferInstallCmd(c *cli.Context) error {
+	if c.NArg() != 1 {
+		return cliutils.WrongNumberOfArgumentsHandler(c)
+	}
+	// Get the artifactory server
+	serverDetails, err := coreConfig.GetSpecificConfig(c.Args()[0], false, true)
```

Comment:
Suggestion - print an error and exit if the server URL does not contain `127.0.0.1` or `localhost`

Reply:
do we need to constraint it to localhosts only? we already have a check that makes sure we can find the plugin directory in the machine, if it won't find it will exit with an error that suggest that it must run on a machine with Artifactory.